### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by abrowao**
Added Routing and fixed an issue with a JSON object

### CODE REVIEW

File: frameworks/Julia/Jewelia/julia_server.jl [hunk 1]
```diff
@@ -14,25 +14,31 @@ end
...
+    
+        return HTTP.Response(200, headers, body = "Hello, World!")
+    end
+    
+    function jsonSerialization(req::HTTP.Request)
+        headers = [ "Content-Type" => "application/json",
+                    "Server" => "Julia-HTTP",
+                    "Date" => Dates.format(Dates.now(), Dates.RFC1123Format) * " GMT" ]
+    
+        jsonString = """{"Message": "Hello, World!"}"""
```
Comment (nbrady-techempower) [inline frameworks/Julia/Jewelia/julia_server.jl:30]: The requirements for this test are such that you must instantiate an object on each request to then be serialized, not a string.

File: frameworks/Julia/Jewelia/julia_server.jl [hunk 2]
```diff
@@ -41,18 +47,20 @@ HTTP.listen("0.0.0.0" , 8080, reuseaddr = true) do http
         row = first(results)
         dbNumber = row[2]
         jsonString = "{\"id\":$randNum,\"randomNumber\":$dbNumber}"
```
Comment (joanhey) [inline frameworks/Julia/Jewelia/julia_server.jl:49]: And here

File: frameworks/Julia/Jewelia/julia_server.jl [hunk 3]
```diff
@@ -79,18 +87,21 @@ HTTP.listen("0.0.0.0" , 8080, reuseaddr = true) do http
             dbNumber = row[2]
             responseArray[i] = JSON3.read("{\"id\":$randNum,\"randomNumber\":$dbNumber}", jsonObj)
```
Comment (joanhey) [inline frameworks/Julia/Jewelia/julia_server.jl:88]: And here

File: frameworks/Julia/Jewelia/julia_server.jl [hunk 4]
```diff
@@ -120,14 +131,15 @@ HTTP.listen("0.0.0.0" , 8080, reuseaddr = true) do http
             results = DBInterface.execute(conn, sqlQuery)
             responseArray[i] = JSON3.read("{\"id\":$randId,\"randomNumber\":$randNum}", jsonObj)
```
Comment (joanhey) [inline frameworks/Julia/Jewelia/julia_server.jl:132]: And here

**Review by joanhey — CHANGES_REQUESTED** [review]

File: frameworks/Julia/Jewelia/julia_server.jl (hunk 1 above)
Comment (donavindebartolo) [reply frameworks/Julia/Jewelia/julia_server.jl]: I believe this has been addressed now, please verify. Thanks!

**Pull request merged**